src/ResourceManager/AzureBatch/Commands.Batch/help/Get-AzureBatchNodeFileContent.md

```diff
@@ -130,6 +132,32 @@ Accept pipeline input: True (ByValue)
 Accept wildcard characters: False
 ```
 
+### -ByteRangeEnd
+The end of the byte range to be downloaded.```yaml
```

Comment:
@matthchr `platyPS` does this weird thing where this `` ```yaml `` string is added at the end of the description rather than on the next line by itself, which can cause the markdown to be rendered incorrectly (like it does in this file). Would you mind moving all instances of this `` ```yaml `` string to a new line by itself?